### PR TITLE
Remove 'authenticated with sso' param from MHV URL helper

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description
A contact at MHV has expressed to us that we should always be linking to their "eauth" domain, regardless of SSOe status. This PR makes that update to our utility function and removes the unused parameter from the calls to that function.

https://github.com/department-of-veterans-affairs/va.gov-team/issues/17318

## Testing done
Visited health care portals and confirmed that the eauth MHV URLs are being used correctly

### RX
image

### Messaging
image

### Lab tests
image

### Records
image

## Acceptance criteria
- [ ] User always lands on the "eauth" domain

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/dashboard/tests/components/cerner-widgets.unit.spec.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ describe('Prescriptions Widget', () => {
       name: /Go to My HealtheVet/i,
     });
     expect(ctaButton.href).to.contain(
-      'https://mhv-syst.myhealth.va.gov/mhv-portal-web/web/myhealthevet/refill-prescription',
+      'https://int.eauth.va.gov/mhv-portal-web/eauth?deeplinking=prescription_refill',
```
Comment (kelsonic) [inline src/applications/personalization/dashboard/tests/components/cerner-widgets.unit.spec.js:46]: Is this subdomain right? Just double checking 🙂

**Review by kelsonic — APPROVED** [review]
LGTM! 🎉

File: src/applications/personalization/dashboard/tests/components/cerner-widgets.unit.spec.js (hunk 1 above)
Comment (ncksllvn) [reply src/applications/personalization/dashboard/tests/components/cerner-widgets.unit.spec.js:46]: it's the correct environment for dev.va.gov (that's the env our unit tests execute under)

Comment (kelsonic) [reply src/applications/personalization/dashboard/tests/components/cerner-widgets.unit.spec.js:46]: Ahh that makes sense! 🙂

**Review by micahchiang — APPROVED** [review]

File: src/platform/site-wide/mhv/utilities.js [hunk 2]
```diff
@@ -19,13 +18,11 @@ const mhvToEauthRoutes = {
 // 1. Whether this is a production or staging environment
 // 2. Whether the current user is authenticated with SSOe
```
Comment (erikphansen) [inline src/platform/site-wide/mhv/utilities.js:19]: this should be removed/updates

**Review by erikphansen — APPROVED** [review]
Just one small comment update requested

**Review by asg5704 — APPROVED** [review]
LGTM.

**Review by theodur — APPROVED** [review]

**Pull request merged**